Calibration/HcalCalibAlgos/test/isoTrackRecoAnalysis_cfg.py

```diff
@@ -0,0 +1,86 @@
+import FWCore.ParameterSet.Config as cms
+process = cms.Process('ANALYSIS')
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('warnings','errors',
+                                         'cout','cerr'),
+    categories = cms.untracked.vstring('HcalIsoTrack'), 
+    debugModules = cms.untracked.vstring('*'),
+    warnings = cms.untracked.PSet(
+        placeholder = cms.untracked.bool(True)
+    ),
+    default = cms.untracked.PSet(
+
+    ),
+    errors = cms.untracked.PSet(
+        placeholder = cms.untracked.bool(True)
+    ),
+    cerr = cms.untracked.PSet(
+        optionalPSet = cms.untracked.bool(True),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        noTimeStamps = cms.untracked.bool(False),
+        FwkReport = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            reportEvery = cms.untracked.int32(500),
+            limit = cms.untracked.int32(10000000)
+        ),
+        default = cms.untracked.PSet(
+            limit = cms.untracked.int32(10000000)
+        ),
+        Root_NoDictionary = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            limit = cms.untracked.int32(0)
+        ),
+        FwkJob = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            limit = cms.untracked.int32(0)
+        ),
+        FwkSummary = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            reportEvery = cms.untracked.int32(1),
+            limit = cms.untracked.int32(10000000)
+        ),
+        threshold = cms.untracked.string('INFO')
+     ),
+     cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('INFO'),
+        noTimeStamps = cms.untracked.bool(True),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        DEBUG = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        HcalIsoTrack = cms.untracked.PSet(
+            limit = cms.untracked.int32(-1)
+       )
+    )
+)
+
+process.load("Configuration.Geometry.GeometryIdeal_cff")
+process.load('Configuration/StandardSequences/GeometryExtended_cff')
+process.load('Configuration/StandardSequences/MagneticField_38T_cff')
+process.GlobalTag.globaltag = 'START53_V15::All'
```

Comment:
I pushed also the changes as required for global tags in the cfg files we use. We shall keep in mind for the suggestion for geometry and b-field for future - or you want them right now?
Sunanda
________________________________
From: Salvatore Di Guida [notifications@github.com]
Sent: 07 May 2015 17:09
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-alca11 Tune+test several AlCaReco jobs and prepare DQM plots for IsoTrack Trigger (#8943)


In Calibration/HcalCalibAlgos/test/isoTrackRecoAnalysis_cfg.py<https://github.com/cms-sw/cmssw/pull/8943#discussion_r29861224>:

> +        INFO = cms.untracked.PSet(
> +            limit = cms.untracked.int32(0)
> +        ),
> +        DEBUG = cms.untracked.PSet(
> +            limit = cms.untracked.int32(0)
> +        ),
> +        HcalIsoTrack = cms.untracked.PSet(
> +            limit = cms.untracked.int32(-1)
> +       )
> +    )
> +)
> +
> +process.load("Configuration.Geometry.GeometryIdeal_cff")
> +process.load('Configuration/StandardSequences/GeometryExtended_cff')
> +process.load('Configuration/StandardSequences/MagneticField_38T_cff')
> +process.GlobalTag.globaltag = 'START53_V15::All'


@bsunanda<https://github.com/bsunanda>
Thanks for taking care of our comments.
The suggested configuration for geometry:

process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
process.load("Configuration.StandardSequences.MagneticField_cff")


This is the suggested configuration for Global Tag:

process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
from Configuration.AlCa.autoCond import autoCond
process.GlobalTag.globaltag=autoCond['run2_mc'] #use 'run2_data' key in case you want to run on data.


—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/8943/files#r29861224>.